specification/loadtestservice/LoadTestService/routes.tsp

```diff
@@ -144,7 +145,7 @@ numeric, underscore or hyphen characters.
       @body
       body: bytes;
     },
-    FileInfo
+    ResourceCreatedResponse<FileInfo>
```

Comment:
@mikekistler Do we have any guideline for data-plane on returned code(2xx) for various http methods(e.g PUT, PATCH, POST etc)?

Reply:
Guidelines for HTTP response status codes is [here](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md#http-return-codes)